models/comment-model.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../db/connection');
+const moment = require('moment');
+
+exports.getCommentsByArticleQuery = ((id, next) => {
+    return db.query(
+        `SELECT comment_id, votes, created_at, author, body, article_id
+        FROM comments c
+        WHERE article_id = $1
+        ORDER BY created_at DESC;`, [id])
+        
+        .then(({rows}) => {
+            rows.map((article) => {
+                article.created_at = moment(new Date(article.created_at)).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
I am not entirely sure why you are mutating the response you are getting
But apart from that:
- don't use a map as a forEach
- name things appropriately (this isn't an article - you are getting back the comments)
- generally, we don't want to mutate